instance_control/readme.md

```diff
@@ -10,15 +19,15 @@ chmod u+x bubuku_cluster.py
 ```
 
 ### Create
-
+The command is used to create a complete new cluster or add new nodes to the existing cluster (--cluster-size defines how many nodes you want to create/add to the cluster for provided config)
```

Comment:
But actually creating a cluster is not really an action, right? It just happens when the first node is created. So you don't do anything specific for creating a cluster, right?